src/devices/Ina226/README.md

```diff
@@ -0,0 +1,261 @@
+
+## Summary
+
+The INA226 is a current shunt and power monitor with an I2C or SMBUS-compatible
+interface. The device monitors both a shunt voltage drop and bus supply voltage.
+Programmable calibration value, conversion times, and averaging, combined with
+an internal multiplier, enable direct readouts of current in amperes and power
+in watts.
+
+* Senses Bus Voltages from 0 to 36 V
+* High-Side or Low-Side Sensing
+* Reports Current, Voltage, and power
+* High Accuracy: 0.1% Gain Error (Max) 10uv Offset(Max)
+* Configurable Averaging Options
+* 16 Programmable Addresses
+
+## Documentation
+
+* [INA226 Datasheet](https://www.ti.com/lit/ds/symlink/ina226.pdf)
+
+## Potential Smoke Warning
+
+INA226 has a limited common-mode voltage range. This means that if you want to measure, for example solar panel voltage, then you will need a common negative ground charge controller and have the ina226 and shunt connected with a contiguous ground between them and the device connected to the ina226 through i2c. This can be avoided by using an i2c isolator AND a power isolator. There are multiple options for this but one example would be the ADM3260 or similar by other manufacturers.
+
+Also, this board has been modified to add a filter circuit, do not assume that a pre-purchased board will have the same current capability as the one used with the creation of this binding.  Most of them come with .01 ohm resistors which have a small current carrying capability so it has been replaced with a larger shunt of a lower resistance.
+
+## To-Do List
+
+* Fritzing Diagram
+* SMBUS Alert Response (feature has not been implemented)
+* Alert pin related telemetry for the three different alert flags still needs to be tested with hardware.
+* Input validation on current and power properties to prevent out of range exceptions
+
+## Binding Notes
+
+This sample uses a generic INA226 breakout board and is hooked to a generic 100 amp / 75 mv shunt with a .1uf ceramic capacitor across In+/In- and two ten ohm resistors between shunt and In+/In- as a filter circuit. The example prints the different registers and configuration every second until you press ESC. The Alert pin is configured for Over-Voltage fault in this example, meaning that if the bus voltage exceeds the configured bus voltage limit, the alert pin will be triggered.  Configuration allows for a triggered high or triggered low condition and in the example the polarity has been set opposite from default for demonstration purposes.
+
+The calibration is determined by the current least significant bit formula found in the datasheet.  Provide the shunt resistance and the expected max current and SetCalibration() will do the rest.
+
+When the INA226 receives the Reset command, it completely resets registers to default and therefore you need to reconfigure anytime you send a Reset command as well.
+
+```csharp
+
+using Iot.Device.Ina226;
+
+// Default i2c address for the INA226
+const byte INA226_I2cAddress = 0x40;
```

Comment:
you should define const `Ina226.DefaultI2cAddress` and then you don't need a comment once you do